@xen-orchestra/openflow/README.md

```diff
@@ -24,7 +24,7 @@ const version = openflow.versions.openFlow11
 const ofProtocol = openflow.protocols[version]
 
 function parseOpenFlowMessages(socket) {
-  for await (const msg from parse(socket)) {
+  for await (const msg of parse(socket)) {
```

Comment:
Wasn't this corrected in another PR?

Reply:
Yes it was, merged into this branch, the only file to review is `packages/xo-server-sdn-controller/src/index.js`